ext/filter/filter.c

```diff
@@ -615,6 +615,8 @@ static void php_filter_call(zval *filtered, zend_long filter, zval *filter_args,
 
 		if ((option = zend_hash_str_find(HASH_OF(filter_args), "options", sizeof("options") - 1)) != NULL) {
```

Comment:
It would make more sense to use `zend_hash_str_find_deref` here instead.

Reply:
this is a good idea, but `zend_hash_str_find_deref` from 7.2 to support. below 7.2 version is to add the method ?

Reply:
Ah, good point. In that case it would probably be best to keep using `zend_hash_str_find` and perform an explicit `ZVAL_DEREF(option)` inside the if.